File: C Functions Task/todo2/app2.c
#include <stdio.h>
//called function
void Average(int a,int b,int c){
    float result=(a+b+c)/3;
    printf("%f",result);
}

int main()
{
    int a=10;
    int b=20;
    int c=30;
    //function caller
    Average(a,b,c);
    return 0;
}

File: C Functions Task/todo3/app2.java
public class Main
{
	public static void main(String[] args) {
	   	System.out.print("Result:");
		//function caller
	   	Average(10,20,30);
	}
	//called function
	public static void Average(int a,int b,int c){
	    float result=(a+b+c)/3;
	    System.out.println(result);
	}
}
